corehq/apps/accounting/models.py

```diff
@@ -1726,6 +1727,18 @@ def can_reactivate_domain_subscription(cls, account, domain, plan_version,
             last_subscription.plan_version.pk == plan_version.pk
         ), last_subscription
 
+    @property
+    def is_below_minimum_subscription(self):
+        if self.is_trial:
+            return False
+        elif self.date_start < datetime.date(2018, 9, 3):
```

Comment:
This code can be deleted once we're ~30 days past the release date, right?

Reply:
Yeah, that's the plan!

Reply:
excellent